sdk/lib/opentelemetry/sdk/trace/samplers/probability_sampler.rb

```diff
@@ -26,51 +30,48 @@ def initialize(probability, hints:, ignore_parent:, apply_to_root_spans:, apply_
           #
           # Callable interface for probability sampler. See {Samplers}.
           def call(trace_id:, span_id:, parent_context:, hint:, links:, name:, kind:, attributes:)
-            take_hint(hint) ||
-              use_parent_sampling(parent_context) ||
-              use_link_sampling(links) ||
```

Comment:
The OTEP talks about links, but the decision table for `ProbabilitySampler` doesn't use them.